README.md

```diff
@@ -4,5 +4,27 @@ Simplify use of AWS resources in your code with aws-python-utils
 
 ### AwsSecretManager
 
-* get_secret() -- requires ENVIRONMENT variable set "AWS_SECRET_MANAGER_SECRET"
-that refers to the secret key name stored in AWS Secret Manager. Returns secret vaule
+##### get_secret()
+
+Use ENVIRONMENT variable "AWS_SECRET_MANAGER_SECRET" or pass in secret_key name.
+
+###### Example Usage
+
+Via ENVIRONMENT variable
+```
+In [1]: import os
+In [2]: import secretmanager
+In [3]: os.environ["AWS_SECRET_MANAGER_SECRET"] = "mysecret"
```

Comment:
try `os.environ.get('AWS_SECRET_MANAGER_SECRET', 'DEFAULT_NAME')` to avoid npe's

Reply:
Python doesn't have npe's but I think if we just drop the default None it will serve the same purpose. Let me test